pkg/apis/apps/v1/conversion.go

```diff
@@ -144,10 +144,7 @@ func Convert_v1_RollingUpdateDeployment_To_apps_RollingUpdateDeployment(in *apps
 	if err := s.Convert(in.MaxUnavailable, &out.MaxUnavailable, 0); err != nil {
 		return err
 	}
-	if err := s.Convert(in.MaxSurge, &out.MaxSurge, 0); err != nil {
-		return err
-	}
-	return nil
+	return s.Convert(in.MaxSurge, &out.MaxSurge, 0)
 }
```

Comment:
In this file, most of some judgement pattern are copy from auto-generated file, i think it is normal